code/modules/reagents/chemistry/reagents/drinks/drink_reagents.dm

```diff
@@ -0,0 +1,912 @@
+	taste_description = "berries"
+
+/datum/reagent/consumable/poisonberryjuice/on_mob_life(mob/living/L, metabolism)
+	L.adjustToxLoss(1)
+	return ..()
+
+/datum/reagent/consumable/watermelonjuice
+	name = "Watermelon Juice"
+	description = "Delicious juice made from watermelon."
+	color = "#863333" // rgb: 134, 51, 51
+	taste_description = "juicy watermelon"
+
+/datum/reagent/consumable/lemonjuice
+	name = "Lemon Juice"
+	description = "This juice is VERY sour."
+	color = "#863333" // rgb: 175, 175, 0
+	taste_description = "sourness"
+
+/datum/reagent/consumable/banana
+	name = "Banana Juice"
+	description = "The raw essence of a banana. HONK"
+	color = "#863333" // rgb: 175, 175, 0
+	taste_description = "banana"
+
+/datum/reagent/consumable/nothing
+	name = "Nothing"
+	description = "Absolutely nothing."
+	taste_description = "nothing"
+
+/datum/glass_style/shot_glass/nothing
+	required_drink_type = /datum/reagent/consumable/nothing
+	icon_state = "shotglass"
+
+/datum/reagent/consumable/laughter
+	name = "Laughter"
+	description = "Some say that this is the best medicine, but recent studies have proven that to be untrue."
+	custom_metabolism = INFINITY
+	color = "#FF4DD2"
+	taste_description = "laughter"
+
+/datum/reagent/consumable/laughter/on_mob_life(mob/living/carbon/M)
+	M.emote("laugh")
+	return ..()
+
+/datum/reagent/consumable/superlaughter
+	name = "Super Laughter"
+	description = "Funny until you're the one laughing."
+	custom_metabolism = 1.5 * REAGENTS_METABOLISM
+	color = "#FF4DD2"
+	taste_description = "laughter"
+
+/datum/reagent/consumable/potato_juice
+	name = "Potato Juice"
+	description = "Juice of the potato. Bleh."
+	nutriment_factor = 2 * FOOD_METABOLISM
+	color = "#302000" // rgb: 48, 32, 0
+	taste_description = "irish sadness"
+
+/datum/reagent/consumable/pickle
+	name = "Pickle Juice"
+	description = "More accurately, this is the brine the pickle was floating in"
+	nutriment_factor = 2
+	color = "#302000" // rgb: 48, 32, 0
+	taste_description = "vinegar brine"
+
+/datum/reagent/consumable/grapejuice
+	name = "Grape Juice"
+	description = "The juice of a bunch of grapes. Guaranteed non-alcoholic."
+	color = "#290029" // dark purple
+	taste_description = "grape soda"
+
+/datum/reagent/consumable/plumjuice
+	name = "Plum Juice"
+	description = "Refreshing and slightly acidic beverage."
+	color = "#b6062c"
+	taste_description = "plums"
+
+/datum/reagent/consumable/milk
+	name = "Milk"
+	description = "An opaque white liquid produced by the mammary glands of mammals."
+	color = "#DFDFDF" // rgb: 223, 223, 223
+	taste_description = "milk"
+
+/// Simple healing proc to avoid copy paste
+/datum/reagent/consumable/proc/milk_heal(mob/living/L, metabolism)
+	L.heal_limb_damage(0.2)
+	if(holder.has_reagent(/datum/reagent/consumable/capsaicin))
+		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
+
+/datum/reagent/consumable/milk/on_mob_life(mob/living/L, metabolism)
+	milk_heal(L, metabolism)
+	return ..()
+
```

Comment:
erm?

Reply:
This is just consolidation of all the 0.2 brute heal that all milk and milk childed reagents have. Seems TG did that as they separated the childed reagents into their own, all this is. Dunno if it'd be better some other way.